eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelExceptions.scala

```diff
@@ -78,6 +78,7 @@ case class ExpiryTooBig                            (override val channelId: Byte
 case class HtlcValueTooSmall                       (override val channelId: ByteVector32, minimum: MilliSatoshi, actual: MilliSatoshi) extends ChannelException(channelId, s"htlc value too small: minimum=$minimum actual=$actual")
 case class HtlcValueTooHighInFlight                (override val channelId: ByteVector32, maximum: UInt64, actual: MilliSatoshi) extends ChannelException(channelId, s"in-flight htlcs hold too much value: maximum=$maximum actual=$actual")
 case class TooManyAcceptedHtlcs                    (override val channelId: ByteVector32, maximum: Long) extends ChannelException(channelId, s"too many accepted htlcs: maximum=$maximum")
+case class DustHtlcExposureTooHighInFlight         (override val channelId: ByteVector32, maximum: Satoshi, actual: MilliSatoshi) extends ChannelException(channelId, s"dust htlcs hold too much value: maximum=$maximum actual=$actual")
```

Comment:
A code nit is that it would force us to have a bit more code in `receiveFee` / `sendFee` to send the right error.
If you think it's important to distinguish the two cases I can try though.

Reply:
Actually it's not too bad: done in https://github.com/ACINQ/eclair/pull/1985/commits/65c02c3aa61bceb358660a305dc07bc28030ad09